### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Iptysam**
Update index.html
Update style.css

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,1990 +1,4132 @@
...
+  .left_block2,
+  .left_block3,
+  .navmenu,
+  .menu,
+  .portfolio,
+  .about,
+  .Contact,
+  .handcursor {
+    display: none;
+  }
```
Comment (Kamba56) [inline style.css:2089]: - [ ] I observed that the details in the work cards disappear when switched to the mobile version. Please fix this as you should stick to the design in desktop and mobile versions.

## Your design
d1

## Figma template
ccc
b15ff44.png)

File: style.css [hunk 2]
```diff
@@ -1,1990 +1,4132 @@
...
+
+  .header {
+    display: flex;
+    position: absolute;
+    left: 0;
+    right: 0;
+    top: 44px;
+    bottom: 0;
+  }
```
Comment (Kamba56) [inline style.css]: - [x] Please remove the space at the top of your mobile header to match the figma template.
## Your design
d2
## Figma template
aaa

File: style.css [hunk 3]
```diff
@@ -1,1990 +1,4132 @@
...
+    height: 846px;
+    left: 1px;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    z-index: 10;
+    background: url(Images/contactformbackgroundshapedesktop.png);
+    background-position: 50% 50%;
+    background-size: cover;
+  }
```
Comment (Kamba56) [inline style.css]: - [ ] I noticed a space below your footer, Kindly remove the space so your portfolio will match the design template.
- [ ] Please center your contact form to match the design template.
## Your design
d3
## Figma template
zzz

- [Optional] Consider removing all static heights and widths. It is best in responsive design not to use static heights and widths. You can use `height: auto;` and `width` in percentage `%` instead.

**Review by Kamba56 — COMMENTED** [review]
Hi @Iptysam, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] All linters passed
- [x] Correct use of GIthub flow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -64,36 +77,53 @@ <h3 class="Proj2">Multi-Post Stories</h3>
...
             <div class="container3">
                 <div class="rect3"></div>
-                <h4 class="Proj3">Tonic</h4>
+                <div class="left_block2">
+                <h4 class="Proj3a">Tonic</h4>
+                <h4 class="Proj3">Facebook 360</h4> 
+                <ul class="list1a">
+                <li class="frame2a"><img src="Images/Frame2.png" alt="frame"></li>
+                </ul>
+                <p class="text1a">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
```
Comment (tufoinnkuo10) [inline index.html:91]: - [optional ] Great job making the changes. Please fix this extra right margin in both your mobile and desktop right margin
Screenshot (345)_LI
Screenshot (347)_LI

**Review by tufoinnkuo10 — COMMENTED** [review]
## Status: Approved :heavy_check_mark:

Hi @Iptysam   

Your project is complete! Great job for making the requested changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**